packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -3,12 +3,14 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:math';
 
 import 'package:camera/camera.dart';
 import 'package:camera_platform_interface/camera_platform_interface.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:pedantic/pedantic.dart';
```

Comment:
do we need this dependency? Seems it shouldn't be here.

Reply:
Yes, we need it for `unawaited` on line 212. It is [recommended](https://dart-lang.github.io/linter/lints/unawaited_futures.html) to do so by the dart linter.

Reply:
Then we need to change dependency from a `dev_dependency` to a normal `dependency` in the pubspec.yaml

Reply:
Good catch! Moving it.